sbat.c

```diff
@@ -120,4 +123,161 @@ EFI_STATUS parse_sbat(char *sbat_base, size_t sbat_size, char *buffer,
 	return efi_status;
 }
 
+EFI_STATUS
+verify_sbat(struct sbat *sbat, list_t *sbat_entries)
+{
+	unsigned int i;
+	list_t *pos = NULL, *tmp = NULL;
+
+	for (i = 0; i < sbat->size; i++) {
+		struct sbat_entry *entry = entry = sbat->entries[i];
+		for_each_sbat_var_safe (pos, tmp, sbat_entries) {
```

Comment:
The `_safe` variant is only needed when you will modify the list / remove elements, but just to iterate you can use the `for_each_sbat_var()` variant.